jupyterhub_moss/spawner.py

```diff
@@ -96,12 +101,17 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.options_form = self.create_options_form
 
-    def _get_slurm_info(self):
+    async def _get_slurm_info(self):
```

Comment:
👍 

Could you add `batchspawner>=1.0` in `pyproject.toml` for the switch to `async`?

Reply:
good point, done in https://github.com/silx-kit/jupyterhub_moss/pull/64/commits/f1a874dc7f18f1e3a43ce285de1fef5e12cd9497